content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -146,6 +146,11 @@ The following plugins support `WaitForFirstConsumer` with dynamic provisioning:
 * [GCEPersistentDisk](#gce-pd)
 * [AzureDisk](#azure-disk)
 
+{{< feature-state state="beta" for_k8s_version="1.14" >}}
```

Comment:
From the preview, it kind of looks like this feature state applies to the rest of the section, not just the CSI part.  Does anyone have suggestions on how to make less misleading?

Reply:
maybe move lines 154 to 158 before the shortcode? Does that make it clear enough?

Reply:
I tried moving CSI to its own section and rewording.